analysis/bin/BuildFile.xml

```diff
@@ -33,11 +33,18 @@
   <flags CXXFLAGS="-g" />
 </bin>
 
+<<<<<<< HEAD
```

Comment:
You need to fix this when you pull in from master. Basically you chose to take one or the other, or merge them together manually

Reply:
From <<<<<< to ====== is one version, the other is from ======= to >>>>>>>